velox/dwio/dwrf/common/wrap/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# These files wrap the generated source and header files from
+# velox_dwio_dwrf_proto with pragamas to disable certain warnings TODO: transfer
+# disabling the warnings to CMake/Buck
+velox_add_library(velox_dwio_dwrf_proto orc-proto-wrapper.cpp
```

Comment:
Did we not need this before ?

Reply:
They were built twice and linked inconsistently so I unified this. I did not test if we actually still need the pragmas.